src/openforms/templates/authentication/registrator_subject_info.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'ui/views/abstract/detail.html' %}
```

Comment:
we'll tackle styling this in another PR - I'll also check with Joeri whether we want this to have public UI or admin styling.